### DISCUSSION TITLE
getting error while executing roles in ansible

### DISCUSSION THREAD

**Discussion by vikramdevopsb79** [learndevopsonline/learndevopsonline#1907] [Q&A] [answered]
![Screenshot (422)](https://github.com/learndevopsonline/learndevopsonline/assets/169573156/d67cce07-a76f-492c-8cfc-0c56c23530a0)
while executing the given command getting error , i tried to debug things but still issue is not solved error is getting some where else wrong error message getting .

This is my code :  https://github.com/vikramdevopsb79/expense-ansible.git

**Comment by chaithanya1812** [comment, marked as answer]
Hi @vikramdevopsb79 
## Isuue:-  capitial letter ' F '   
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/4abc186a-80a9-44b1-b14c-ade74234795c)
